bin/create_release.sh

```diff
@@ -60,8 +68,14 @@ export RELEASE_URL=$(curl \
     | sed -e 's/.*\(https.*\)\"\,/\1/' \
     | sed -e 's/api/uploads/')
 
-# Release Package
-echo $RELEASE_URL
+# Release URL is created only if there is valid github token
+if [[ ! $RELEASE_URL ]]; then
+  echo "RELEASE_URL is empty. Exiting"
```

Comment:
Just a bit more info here. Something like "This may be caused by an invalid 'GITHUB_TOKEN'."

Reply:
added